### PR TITLE
Blog - Load Testing Backend Services Using K6

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
@balapriyac  @jalletto 

[Preview Link](https://642ad78788895510a66a413d--ephemeral-narwhal-3ac3b2.netlify.app/blog/load-testing-using-k6/)

File: blog/_posts/2023-05-20-load-testing-using-k6.md [hunk 1]
```diff
@@ -0,0 +1,528 @@
...
+sudo mysql
+~~~
+
+Then run the following command to create a new database called `blogFastAPI`.
+
+~~~{.bash caption=">_"}
+
+CREATE DATABASE blogFastAPI;
+
+~~~
```
Comment (balapriyac) [inline blog/_posts/2023-05-20-load-testing-using-k6.md:208]:
```suggestion

~~~{.bash caption=">_"}
CREATE DATABASE blogFastAPI;
~~~
```

File: blog/_posts/2023-05-20-load-testing-using-k6.md [hunk 2]
```diff
@@ -0,0 +1,528 @@
...
+
+CREATE TABLE user_info(
+  id INT(6) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+  username VARCHAR(50) NOT NULL,
+  password VARCHAR(500) NOT NULL,
+  fullname VARCHAR(50) NOT NULL
+);
+
+~~~
+
```
Comment (balapriyac) [inline blog/_posts/2023-05-20-load-testing-using-k6.md:259]:
```suggestion

~~~{.bash caption=">_"}
CREATE TABLE blog (
  id INT(6) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
  title VARCHAR(500) NOT NULL,
  content VARCHAR(5000) NOT NULL
);


CREATE TABLE user_info(
  id INT(6) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
  username VARCHAR(50) NOT NULL,
  password VARCHAR(500) NOT NULL,
  fullname VARCHAR(50) NOT NULL
);

~~~

```

File: blog/_posts/2023-05-20-load-testing-using-k6.md [hunk 3]
```diff
@@ -0,0 +1,528 @@
...
+Navigate back to the terminal that is currently inside the root project directory of the app. Inside the file `database.py`, you have the codes to help the blog service access the MySQL database.
+
+~~~{.bash caption=">_"}
+
+USER = os.getenv('DB_USER')
+PASSWORD = os.getenv('DB_PASSWORD')
+
+SQLALCHEMY_DATABASE_URL = (f"mysql+mysqlconnector://{USER}:{PASSWORD}@localhost:3306/blogFastAPI")
+~~~
+
```
Comment (balapriyac) [inline blog/_posts/2023-05-20-load-testing-using-k6.md:273]:
```suggestion

~~~{.bash caption=">_"}
USER = os.getenv('DB_USER')
PASSWORD = os.getenv('DB_PASSWORD')

SQLALCHEMY_DATABASE_URL = (f"mysql+mysqlconnector://{USER}:{PASSWORD}@localhost:3306/blogFastAPI")
~~~

```

**Review by balapriyac — CHANGES_REQUESTED** [review]
Thanks Sameer. Suggested a few formatting edits.

**Comment by msali123** [comment]
Changes done @balapriyac 
[Preview Link](https://642d55f521493d1559909d6b--ephemeral-narwhal-3ac3b2.netlify.app/blog/load-testing-using-k6/)

**Comment by balapriyac** [comment]
@jalletto this is ready to merge.

**Review by jalletto — APPROVED** [review]

**Pull request merged**